tracer/test/test-applications/debugger/Samples.Probes/Program.cs

```diff
@@ -1,13 +1,11 @@
 using System;
 using System.Reflection;
-using System.Security.Policy;
-using System.Threading;
 using System.Threading.Tasks;
 using Samples.Probes;
 
 public static class Program
 {
-    private const int millisecondsToWaitSetProbes = 1000 * 4;
+    private const int millisecondsToWaitSetProbes = 1000 * 60;
```

Comment:
Seems like an awful long time to wait? Do we really need this wait (I thought that this code path was made deterministic by waiting for a specific text to show up in our log file?